vernac/egramcoq.ml

```diff
@@ -231,7 +233,7 @@ let adjust_level assoc from = let open Gramlib.Gramext in function
     | _ -> Some None
     end
   (* None means NEXT *)
-  | (NextLevel,_) -> Some None
+  | (NextLevel,_) -> (* invariant custom=custom' expected *) Some None
```

Comment:
Why not an `assert` then?

Reply:
Indeed, the cost is anecdotic, why not. I made the change.